features/302-auth-info-in-code-code/auth-info-in-cloud-code.md

```diff
@@ -0,0 +1,91 @@
+# AuthInfo in cloud code
+
+## Overview
+
+The gateway must resolve the HTTP header `x-skygear-access-token` into `x-skygear-auth-info`
+when the request has been determined to be routed to cloud code.
+The cloud code runtime must decode `x-skygear-auth-info` and
+provide it as a value to the underlying application.
+How it is provided should follow language and framework convention.
+The cloud code SDK must provide ways to decode it.
+
+## x-skygear-auth-info
+
+This is a HTTP Header containing a [AuthInfo](#AuthInfo) encoded with [base64url](#base64url)
+
+Example
+
+```
+x-skygear-auth-info: eyJfaWQiOiAiODdkZmFhY2YtYTg3Mi00NDRhLTk0OGEtMTQ5N2M2YmIyYTAzIiwgImRpc2FibGVkIjogZmFsc2UsICJ2ZXJpZmllZCI6IGZhbHNlfQ==
+```
+
+Note that this representation should be treated as implementation details and subject to changes.
+In the future, signature may be added so that this header can be verified.
+
+If `x-skygear-access-token` is absent, then `x-skygear-auth-info` is absent as well.
+Therefore, the provided value must use [Option Type](https://en.wikipedia.org/wiki/Option_type).
+
+## AuthInfo
+
+This is a JSON Object of the following fields.
+
+- `_id`: string
```

Comment:
Do we still using `_id`? Better to sync with the user response of auth gear. 

In this spec, we are using `user_id`
https://github.com/SkygearIO/features/pull/254/files